src/router/client.rs

```diff
@@ -495,7 +496,7 @@ impl RouterClient {
         StateChannelMessage::offer(
             packet.packet().clone(),
             self.keypair.clone(),
-            self.region,
+            &self.region,
```

Comment:
Minor nit: region isn’t Copy, so why this change?

Reply:
Because the offer call only needs a reference to it to construct the representing internal value